### DISCUSSION TITLE
According the tutorial, i fail at this step. I do not know why. i see the function.

### DISCUSSION THREAD

**Discussion by iganggang** [autogluon/autogluon#4156] [General]
![image](https://github.com/autogluon/autogluon/assets/67415290/bde88a83-a81b-4ef7-a871-8e2a11a91631)

**Comment by shchur** [comment]
Hi @iganggang, this method was only added in the latest release (v1.1) and it seems that your machine has an older version of AutoGluon installed. Please upgrade AutoGluon to v1.1 using, e.g., the following command
```
pip install -U autogluon
```
